tests/README.md

```
@@ -0,0 +1,193 @@
+### Quick Start with Docker
+
+1. Start a Vaultwarden instance:
+
+```bash
+docker run -d --name vaultwarden-test \
+  -p 8080:80 \
+  -e SIGNUPS_ALLOWED=true \
+  vaultwarden/server:latest
+```
+
+2. Wait for it to be ready (~10 seconds)
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)